cypress/support/fragments/settings/tenant/location-setup/locations.js

```diff
@@ -84,8 +84,8 @@ export default {
     LocationDetails.waitLoading();
     LocationDetails.checkActionButtonAbsent();
   },
-  selectInstitution() {
-    cy.do(Select('Institution').choose(including('KU')));
+  selectInstitution(content) {
+    cy.do(Select('Institution').choose(including(content)));
```

Comment:
please if you change methods, make sure it doesn't break other tests

Reply:
Yes, I`ve already changed the method. thanks!